admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```diff
@@ -1338,6 +1338,33 @@ $product-page-padding-bottom: 80px !default;
       color: $brand-primary;
     }
   }
+
+  .tokenfield {
+    padding: 0.15rem;
+
+    .token {
+      padding: 0.4rem 0.7rem;
+      background-color: $gray-medium;
+      @include border-radius(0.25rem);
+
+      .close {
+        float: left;
+        margin: 0 0.4rem 0 0;
+        font-size: 1.5rem;
+        line-height: 1rem;
+        color: $white;
+        opacity: 1;
+
+        &:hover {
+          opacity: 0.5;
+        }
+      }
+    }
+
+    .token-input {
+      min-width: 768px;
```

Comment:
This thing will break the mobile layout, please don't!!

You should try to avoid it somehow, maybe a `width: 100%; max-width: 768px;` would do the job, overwise you need to find another way to do this